src/main/java/analyzer/AnnotatorAnalyzer.java

```diff
@@ -1,14 +1,13 @@
 package analyzer;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 import dataObject.Author;
 import dataObject.FileInfo;
 import dataObject.LineInfo;
 import dataObject.RepoConfiguration;
 
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
-
 /**
  * Created by matanghao1 on 5/8/17.
```

Comment:
Remove these too? We don't usually mention author or date in header comments

Reply:
Okay